### ISSUE TITLE
Singleplayer issues

### ISSUE THREAD

**Issue by Shifulor** [R2Northstar/Northstar#218]
The game always stopd when playing single player and getting any achivement. I hsve had tris issue many, many times and it always happens when ypu get an achievement

**Comment by Calyxman** [comment]
Can confirm. Tried to play the level after BT-7274, got 'Clean Shaven' (the achievement where you save both militia tones in the level) and then got hit with some sort of rocket velocity issue and a crash.

**Comment by Shifulor** [comment]
Yeah, saving works and after a restart it runs.. until the next achievement :( Kinda annoying to have 50 crashes, but i believe this issue is related to oringin being origin. Are you on Linux?
